instrumentation/servlet/servlet-common/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/servlet/ExperimentalSnippetHolder.java

```diff
@@ -5,15 +5,23 @@
 
 package io.opentelemetry.javaagent.bootstrap.servlet;
 
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.incubator.config.internal.DeclarativeConfigUtil;
 import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
 
 public class ExperimentalSnippetHolder {
 
   private static volatile String snippet = getSnippetSetting();
 
   private static String getSnippetSetting() {
-    String result = ConfigPropertiesUtil.getString("otel.experimental.javascript-snippet");
-    return result == null ? "" : result;
+    OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
+    return DeclarativeConfigUtil.getInstrumentationConfig(openTelemetry, "servlet")
+        .getString(
+            "javascript_snippet/development",
+            // use ConfigPropertiesUtil only to prevent that the deprecated property is used in
+            // declarative config
```

Comment:
i'm not sure what this comment is trying to say

Reply:
realized that I can just use SPECIAL_MAPPINGS instead